PERSONAL_DATA_DISCLOSURE.md

```diff
@@ -44,7 +44,10 @@ For the purposes of this form, "store" includes the following:
 
 ## Privacy Laws, Regulations, and Policies
 The following laws and policies were considered when creating the list of personal data fields above.
```

Comment:
I'm fine with these changes, however, it sort of renders this description inaccurate. 
```suggestion
Numerous laws and policies were considered when creating the list of personal data fields above.  For additional information, please refer to the following:
```

Thoughts?